CNN-PDE.ipynb

```diff
@@ -0,0 +1,311 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "time it takes is 99.19900250434875\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Run 1000 steps of PDE\n",
+    "start = time.time()\n",
+    "for i in range(300):\n",
+    "    # Step simulation\n",
+    "    steps.run({eps: 0.03, damping: 0.04, c: 3.0})\n",
+    "    DisplayArray(U.eval(), rng=[-0.1, 0.1])\n",
+    "    m= U.eval()\n",
+    "    DisplayArray(m, rng=[-0.1, 0.1])\n",
```

Comment:
here you also time the display, which as others figured out takes most of the time (eg: factor 4 on Raspi4)